example/trace/http/go.mod

```diff
@@ -11,7 +11,7 @@ replace github.com/GoogleCloudPlatform/opentelemetry-operations-go => ../../..
 replace github.com/GoogleCloudPlatform/opentelemetry-operations-go/internal/cloudmock => ../../../internal/cloudmock
 
 require (
-	github.com/GoogleCloudPlatform/opentelemetry-operations-go v0.34.1
+	github.com/GoogleCloudPlatform/opentelemetry-operations-go v1.8.0
```

Comment:
is this really supposed to go from `v0.34.1` to `v1.8.0`?

Reply:
no.  i've filed https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/issues/520 to address that.